_theme_dev/src/css/theme/components/_mobile-menu.scss

```diff
@@ -0,0 +1,66 @@
+
+.mobile-menu {
```

Comment:
I'd consider something less generic there

Reply:
Thanks for suggestion :). Changes applied.